### PR TITLE
✨Group Templates by Tag

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

In the sim4life-lite product, the Templates (aka Tutorials) need to be grouped by Tag so that users don't get a 40 items flat list.

To achieve this, we will take advantage of the recently implemented tags-for-groups feature. The idea is to tag the publicly available templates with tags that are also available to Everyone and use them for grouping the Tutorials.

If a tutorial has multiple tags, it will be present in multiple groups. On the other hand, if it has no tags, it will fall in the "No Group" group.

Main changes:
- New container SlideBar based implemented. The container works similar to a carousel with partially transparent buttons on both sides.
- All Sorting, View and Grouping buttons are now on the right side of the secondary bar. The rest of the buttons that were part of the secondary bar (Select studies, Delete studies) were moved to the left side.
- sim4life-lite product will list the Tutorials grouped by default
- Templates are all fetched before displaying them

The grouping can be extended to more types like Last changed (this week, last month, last year), Shared with (everyone, organizations, private)... this can be implemented in the next iterations

Tutorials grouped:
Groping

## Related issue/s

Templates/Tutorials grouping: https://github.com/ITISFoundation/osparc-issues/issues/765

### CODE REVIEW

**Comment by pcrespov** [comment]
@odeimaiz  is this really for review? If so, can you remove the WIP and add some description to it please

File: api/specs/common/schemas/project-v0.0.1-converted.yaml [hunk 1]
```diff
@@ -407,7 +407,9 @@ properties:
                   - 0
                   - 2
               instructions:
-                type: string
+                type:
+                  - string
```
Comment (pcrespov) [inline api/specs/common/schemas/project-v0.0.1-converted.yaml:411]: these changes are in the other PR, right?

Comment (odeimaiz) [reply api/specs/common/schemas/project-v0.0.1-converted.yaml:411]: Yes https://github.com/ITISFoundation/osparc-simcore/pull/3593

**Comment by odeimaiz** [comment]
> @odeimaiz is this really for review? If so, can you remove the WIP and add some description to it please

It is not, it's like two days away.

**Comment by mrnicegyu11** [comment]
supernice, thx

**Review by pcrespov — APPROVED** [review]
:tada: looks great!

**Review by sanderegg — COMMENTED** [review]
magnifico!

**Review by mguidon — APPROVED** [review]
Real nice!

**Review by GitHK — APPROVED** [review]
Very nice!

**Pull request merged**